code/modules/food_and_drinks/food/snacks_soup.dm

```diff
@@ -124,6 +124,15 @@
 	tastes = list("tomato" = 1, "mint" = 1)
 	foodtype = VEGETABLES | MEAT
 
+/obj/item/reagent_containers/food/snacks/soup/clownchili
+	name = "chili con carnival"
+	desc = "A delicious stew of meat, chiles, and salty, salty clown tears."
+	icon_state = "clownchili"
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/tomatojuice = 2, /datum/reagent/consumable/nutriment/vitamin = 2, /datum/reagent/consumable/laughter = 1)
+	list_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/capsaicin = 1, /datum/reagent/consumable/tomatojuice = 2, /datum/reagent/consumable/nutriment/vitamin = 2, /datum/reagent/consumable/banana = 1)
+	tastes = list("tomato" = 1, "hot peppers" = 2, "clown feet" = 2, "kind of funny" = 2, "someone's parents" = 2)
```

Comment:
:thinking:

Reply:
That clown had a family

Reply:
I changed this to say "a hint of childhood trauma".